java/src/jmri/jmrix/easydcc/EasyDccConsistManager.java

```diff
@@ -53,12 +54,12 @@ public boolean csConsistNeedsSeperateAddress() {
      * consistTable/consistList
      */
     @Override
-    public Consist addConsist(DccLocoAddress address) {
+    public Consist addConsist(LocoAddress address) {
         if (consistTable.containsKey(address)) { // no duplicates allowed.
             return consistTable.get(address);
         }
         EasyDccConsist consist;
-        consist = new EasyDccConsist(address);
+        consist = new EasyDccConsist((DccLocoAddress) address);
```

Comment:
Before this cast, you need to make sure the address is a DCCLocoAddress.  You should throw an illegal argument exception if it isn't.